person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Setters
+
+  # Getters
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def of_age
+    @age >= 18
+  end
+  private :of_age
+
+  def can_use_services
+    of_age | @parent_permission
```

Comment:
- [ ] This boolean operation is used rareil. I would suggest you change `|` to common case `||` to make your code better.

Explanation:

> | - (Condition 1 | Condition 2): checks both cases even if case 1 is true
> 
> || - (Condition 1 || Condition 2): doesn't bother to check the second case if the first one is true
> 
> || - operator will make your code run faster, professionally | is rarely used